components/features/dashboard/student/studentProfile/index.jsx

```diff
@@ -543,7 +545,7 @@ export const ProfilePicture = ({ user }) => {
                   ? user?.user?.profilePhotoUrl
                   : `/assets/img/features/dashboard/student/woman.png`
               }
-              alt="Flowers"
+              alt="ProfilePix"
               style={{
```

Comment:
While you were resolving the conflicts, the hero page image has been tampered with. Kindly fix that ASAP.

## Your design:
![image](https://user-images.githubusercontent.com/10332499/202106216-61a0451a-7d8c-4386-a49f-f2943b24db6f.png)


## Current site:
![image](https://user-images.githubusercontent.com/10332499/202106353-ddb089c3-2672-4a54-8456-200f9be956ee.png)


Please make that adjustment and revert ASAP.